openstack/blockstorage/extensions/volumeactions/results.go

```diff
@@ -50,6 +55,101 @@ func (r commonResult) Extract() (map[string]interface{}, error) {
 	return s.ConnectionInfo, err
 }
 
+// VolumeImage contains information about volume upload to an image service.
```

Comment:
`VolumeImageVolumeType`

(side note: I'm not a fan of this name, but I can't think of a better one. Suggestions are welcomed, or else we'll just use what's here)